apps/cowswap-frontend/src/modules/hooksStore/pure/AddCustomHookForm/CustomDappLoader/index.tsx

```diff
@@ -15,6 +18,20 @@ interface ExternalDappLoaderProps {
   setManifestError: Dispatch<SetStateAction<string | React.ReactNode | null>>
 }
 
+const TIMEOUT = 5000
+
+const fetchWithTimeout = async (url: string, options: any) => {
+  try {
+    const response = await fetch(url, { signal: getTimeoutAbortController(options.timeout).signal, ...options })
+    return response
+  } catch (error) {
+    if (error.name === 'AbortError') {
+      throw new Error(ERROR_MESSAGES.TIMEOUT)
+    }
+    throw error
+  }
+}
```

Comment:
this function looks pretty generic to me. would it make sense to move it to '@cowprotocol/common-utils' so its closer to `getTimeoutAbortController`

Reply:
Refactored now